### ISSUE TITLE
Split long messages

### ISSUE THREAD

**Issue by dec0dOS** [ivanmarban/winston-telegram#29]
Hello, thanks for the awesome project!

I have an issue with the long messages as all unhandled exceptions from the server are sent to telegram.

For example:
```js
logger.error("1".repeat(5000));
```
Logger.error is used for logging messages to telegram. 

This line throws the following exception:
```js
[error] uncaughtException: Unhandled error. ('400: Bad Request: message is too long')
Error [ERR_UNHANDLED_ERROR]: Unhandled error. ('400: Bad Request: message is too long')
 at new NodeError (node:internal/errors:371:5)
 at DerivedLogger.emit (node:events:383:17)
 at DerivedLogger.transportEvent (/project-path/node_modules/winston/lib/winston/logger.js:630:12)
 at Telegram.emit (node:events:406:35)
 at /project-path/node_modules/winston-telegram/lib/winston-telegram.js:133:14
 at IncomingMessage.<anonymous> (/project-path/node_modules/winston-telegram/lib/winston-telegram.js:155:9)
 at IncomingMessage.emit (node:events:406:35)
 at endReadableNT (node:internal/streams/readable:1343:12)
 at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

The message should be split after 4096 characters to fix unexpected logging exceptions.

For now, I'm using the following technique to cut the message:
```js
    formatMessage: function (options) {
      let message = options.message;
      if (message.length > 4000) {
        message = message.substring(0, 4000);
      }
      return message;
    },
```

**Comment by ivanmarban** [comment]
Do you really need to send unhandled exceptions to telegram ?

Maybe you can configure the logger to not send them to telegram and configure another logger to take care of them ?

Something similar to this: https://github.com/winstonjs/winston#exceptions

**Comment by dec0dOS** [comment]
@ivanmarban, thanks a lot for the quick response!

Yes, that is the case. I need to send some debugging data also. I expect that message splitting is handled on the library side but not on the main application level.

That would be awesome if the message splitting feature could be implemented in winston-telegram.

**Comment by dec0dOS** [comment]
Logging to the console or the file works as expected, by the way. The problem is when the long message is sent to the telegram.

**Comment by ivanmarban** [comment]
I will try to implement the functionality to automatically split messages longer than 4096 characters as soon as my free time let me.

Thanks for your feedback !

**Comment by ivanmarban** [comment]
BTW any help is welcome of course.

At first sight my idea is to make a function that wraps around 'send'  to add the split functionality.

**Comment by dec0dOS** [comment]
Splitting for chunks function could be something like this:
```js
function chunkString (str, len) {
  const size = Math.ceil(str.length/len);
  const arr = Array(size);
  let offset = 0;
  
  for (let i = 0; i < size; i++) {
    arr[i] = str.substr(offset, len);
    offset += len;
  }
  
  return arr;
}
```
Source: https://stackoverflow.com/a/14349616

Seems that it is one of the fastest and the simplest solutions to split the string into chunks.

**Comment by ivanmarban** [comment]
@dec0dOS could you please give a try at the code on _**feature/split-long-messages**_ branch and give some feedback ?

**Comment by dec0dOS** [comment]
@ivanmarban, it works as expected in all my use cases. Thanks a lot!

**Comment by dec0dOS** [comment]
@ivanmarban, is it going to be merged into the main branch?

**Comment by ivanmarban** [comment]
Released new feature on v2.5.0
